metcalcpy/util/ecnt_statistics.py

```diff
@@ -339,18 +353,10 @@ def calculate_ecnt_rmse_oerr(input_data, columns_names, aggregation=False):
 
         Returns:
             calculated ECNT_RMSE_OERR as float
-            or None if some of the data values are missing or invalid
+            or None if some data values are missing or invalid
     """
-    warnings.filterwarnings('error')
-    try:
-        total = get_total_values(input_data, columns_names, aggregation)
-        mse_oerr = sum_column_data_by_name(input_data, columns_names, 'mse_oerr') / total
-        rmse_oerr = np.sqrt(mse_oerr)
-        result = round_half_up(rmse_oerr, PRECISION)
-    except (TypeError, ZeroDivisionError, Warning, ValueError):
-        result = None
-    warnings.filterwarnings('ignore')
-    return result
+
+    return weighted_average(input_data, columns_names, 'mse_oerr', aggregation)
```

Comment:
Same comment as above. We need to aggregate the MSE_OERR and the compute the square root of the result.